buildspec_sync.yml

```diff
@@ -61,7 +61,7 @@ phases:
       - export AWS_SESSION_TOKEN=
 
       # Verify the publishing on Public ECR and Dockerhub
-      - './scripts/publish.sh cicd-verify stable public-repo'
+      - './scripts/publish.sh cicd-verify stable public-repo ${AWS_REGION}'
```

Comment:
Wondering, where are we using the 3rd variable `public-repo`?

Reply:
it is just a sign that we are verify our public repo. Do we need to remove it? (Don't want to make it confused for other engineers..

Reply:
Unnecessary variable. Better to remove it. We do have the comment. 

Also, modify the comment that this will only occur when the region is `us-west-2`

Reply:
Update